pkgs/build-support/emacs/wrapper.nix

```diff
@@ -196,12 +197,18 @@ runCommand
     # Wrap emacs and friends so they find our site-start.el before the original.
     for prog in $emacs/bin/*; do # */
       local progname=$(basename "$prog")
+      local autoloadExpression=""
       rm -f "$out/bin/$progname"
+      if [[ "$(basename -- $prog)" == emacs ]] ;then
```

Comment:
```suggestion
      if [[ "$(basename -- $prog)" == emacs ]]; then
```

Reply:
fixed, thanks

Reply:
(I tested this and it worked correctly.)

Reply:
The quotes in this case are not needed since none of the parts contain whitespace, but I can change it if you want.

Reply:
Okay that makes sense.